packages/IMAPClient-Tests.package/ICMockEndpoint.class/README.md

```diff
@@ -0,0 +1 @@
+This is part of the imap server communication mock up.
```

Comment:
I think the term *mock endpoint for testing purposes* would fit better here.

Reply:
I added "offline":
"Mock endpoint for offline testing purposes"